sdk/search/azure-search-documents/azure/search/documents/indexes/_search_index_client.py

```diff
@@ -477,3 +477,162 @@ def get_service_statistics(self, **kwargs):
         kwargs["headers"] = self._merge_client_headers(kwargs.get("headers"))
         result = self._client.get_service_statistics(**kwargs)
         return result.as_dict()
+
+    @distributed_trace
+    def list_aliases(self, **kwargs: Any) -> ItemPaged[SearchAlias]:
+        # type: (**Any) -> ItemPaged[SearchAlias]
+        """List the aliases in an Azure Search service.
+
+        :keyword select: Selects which top-level properties of the skillsets to retrieve. Specified as a
+         list of JSON property names, or '*' for all properties. The default is all
+         properties.
+        :paramtype select: list[str]
+        :return: List of Aliases
+        :rtype: ~azure.core.paging.ItemPaged[~azure.search.documents.indexes.models.SearchAlias]
+        :raises: ~azure.core.exceptions.HttpResponseError
+
+        """
+        kwargs["headers"] = self._merge_client_headers(kwargs.get("headers"))
+        if kwargs.get('select', None):
+            kwargs['select'] = ','.join(kwargs['select'])
+        # pylint:disable=protected-access
+        return self._client.aliases.list(
+            cls=lambda objs: [x for x in objs], **kwargs
+        )
+
+    @distributed_trace
+    def list_alias_names(self, **kwargs: Any) -> ItemPaged[str]:
+        # type: (**Any) -> ItemPaged[str]
+        """List the alias names in an Azure Search service.
+
+        :return: List of alias names
+        :rtype: ~azure.core.paging.ItemPaged[str]
+        :raises: ~azure.core.exceptions.HttpResponseError
+
+        """
+        kwargs["headers"] = self._merge_client_headers(kwargs.get("headers"))
+
+        return self._client.aliases.list(
+            cls=lambda objs: [x.name for x in objs], **kwargs
+        )
+
+    @distributed_trace
+    def get_alias(self, name: str, **kwargs: Any) -> SearchAlias:
+        # type: (str, **Any) -> SearchAlias
+        """
+
+        :param name: The name of the alias to retrieve.
+        :type name: str
+        :return: SearchAlias object
+        :rtype: ~azure.search.documents.indexes.models.SearchAlias
+        :raises: ~azure.core.exceptions.HttpResponseError
+
+        .. admonition:: Example:
+
+            .. literalinclude:: ../samples/sample_index_crud_operations.py
```

Comment:
Assuming this is copy-pasta, but unless there is a sample for aliases, this would point the wrong thing.